### PR TITLE
Invoice template entity

### PR DESCRIPTION
**Pull request by rowinskidamian**

### CODE REVIEW

**Review by KrzKos — APPROVED** [review]
good job! 😃

**Review by ewa-git — APPROVED** [review]
nice:)

**Comment by rowinskidamian** [comment]
Merged InvoiceTemplate entity to main

File: fakturogen-invoice/src/test/java/pl/fakturogen/invoice/service/mapper/InvoiceTemplateMapperTest.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        invoiceTemplateDTO.setDiscountAmount(20.00);
+        invoiceTemplateDTO.setCustomer(customerDTO);
+        invoiceTemplateDTO.setProducts(productDTOList);
+
+        invTempEmpty = new InvoiceTemplate();
+        invTempEmptyDTO = new InvoiceTemplateDTO();
+    }
+
+    @Nested
+    class invoiceTemplateToDTO {
```
Comment (aimiebell) [inline fakturogen-invoice/src/test/java/pl/fakturogen/invoice/service/mapper/InvoiceTemplateMapperTest.java:85]: I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Class names should start with an upper case letter (NM_CLASS_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-class-names-should-start-with-an-upper-case-letter-nm-class-naming-convention).
Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).

**Pull request merged**